homeassistant/helpers/config_entry_flow.py

```diff
@@ -136,7 +136,7 @@ async def async_step_user(
     ) -> Dict[str, Any]:
         """Handle a user initiated set up flow to create a webhook."""
         if not self._allow_multiple and self._async_current_entries():
-            return self.async_abort(reason="one_instance_allowed")
+            return self.async_abort(reason="single_instance_allowed")
```

Comment:
I didn't see that you're changing this string. Then it's ok to do all integrations in one PR for this string.